SofaKernel/framework/framework_test/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ set(SOURCE_FILES
     helper/Quater_test.cpp
     helper/SVector_test.cpp
     helper/io/MeshOBJ_test.cpp
+    helper/io/MeshSTL_test.cpp
+    helper/io/ImagePNG_test.cpp
```

Comment:
Conflict resolution went wrong here. We should not have ImagePNG_test on line 18.
See 67f714f3